addons/udes_stock/models/stock_picking.py

```diff
@@ -533,3 +551,36 @@ def unlink_empty(self):
             to_unlink.unlink()
 
         return self - records
+
+    def un_assign_users(self, skip_users=None):
```

Comment:
I'm unsure on this. But can we use unassign - not sure if it is a word or i have made it up

Reply:
Not very confident with English, trustet Pycharm (it is showing underlined which shows when it's not in English) and googled after and it shows that should be separated, thats why changed the name